store/storetest/team_store.go

```diff
@@ -1463,8 +1495,8 @@ func testResetAllTeamSchemes(t *testing.T, ss store.Store) {
 		SchemeId:    &s1.Id,
 	}
 
-	t1 = (<-ss.Team().Save(t1)).Data.(*model.Team)
-	t2 = (<-ss.Team().Save(t2)).Data.(*model.Team)
+	t1, _ = ss.Team().Save(t1)
```

Comment:
probably makes sense to check the error here, if something goes wrong and return an error, we will be dereferencing a nil pointer 2 lines later.